fips202/keccakf1600.c

```diff
@@ -76,95 +76,21 @@ void KeccakF1600x4_StateXORBytes(uint64_t *state, const unsigned char *data0,
   KeccakF1600_StateXORBytes(state + KECCAK_LANES * 3, data3, offset, length);
 }
 
-#if defined(MLKEM_USE_FIPS202_X2_ASM) && \
```

Comment:
As @mkannwischer pointed out, those zipping functions weren't used anywhere, so let's remove them. If we want to re-introduce functions operating on zipped states, we can consult the git history to get them back.